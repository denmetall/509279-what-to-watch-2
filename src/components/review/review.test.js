import React from 'react';
import Review from './review.jsx';
import renderer from 'react-test-renderer';

it(`Components Review renders correctly`, () => {
  const props = {
    review: {
      comment: ``,
      date: ``,
      rating: null,
      user: {
        name: ``
      }
    }
  };
  const tree = renderer
    .create(<Review {...props}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
