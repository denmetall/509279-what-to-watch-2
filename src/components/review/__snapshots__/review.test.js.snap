// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Review renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-24"
      >
        Invalid date
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  />
</div>
`;
